### PR TITLE
asssigment 2 complete

### PR DESCRIPTION
**Pull request by aayushijain1609**

### CODE REVIEW

File: Assigment2/Assignment2.java [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+        String choice = null;
+    
+        do{
+
+                try (Scanner sc = new Scanner(System.in)) {
+                    System.out.println();
+                    System.out.println("Types of Shape :\n\t1)Circle\n\t2)Recatangle\n\t3)Square\n\t4)Cube\n\t5)Triangle\n\t6)Sphere\n\t7)Cylinder\n\t8)exit the program\n\t");
+                    System.out.print("Enter your choice(in numbers): ");
+                    choice = sc.next();
+                }
```
Comment (yashwantdhakad) [inline Assigment2/Assignment2.java:14]: Can you please add the catch block as well?

File: Assigment2/Assignment2.java [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+                switch (choice) {
+                    case "1":
+                        try {
+                            
+                            Cricle cricle = new Cricle();
+                            cricle.calculation();
+                        } 
+                        catch (Exception e) {
+                        
+                            System.out.println("invaild");
```
Comment (yashwantdhakad) [inline Assigment2/Assignment2.java:24]: Can you please add a proper message for invalid?

File: Assigment2/assignment2/Cricle.java [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+        try {
+
+            double area, perimeter;
+            area = 3.14 * radius * radius;
+            perimeter = 2 * 3.14 * radius;
+
+            System.out.println("Area of Circle: " + area);
+            System.out.println("Perimeter of Circle: " + perimeter);
+        } catch (Exception e) {
+            System.out.println("Error : " + e);
```
Comment (yashwantdhakad) [inline Assigment2/assignment2/Cricle.java:15]: Can you please throw an exception from this method?

File: Assigment2/assignment2/Cube.java [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+            int len = 3;
+            try {
+                double surfacearea,volume;
+                surfacearea = 6 * (len * len);
+                volume = len * len * len;
+                System.out.println("surface area of cube" +surfacearea);
+                System.out.println("volume of cube"+volume);
+            } 
+            catch (Exception e) {
+                System.out.println("error" + e);
```
Comment (yashwantdhakad) [inline Assigment2/assignment2/Cube.java:14]: Similar for this method as well

**Review by yashwantdhakad — COMMENTED** [review]
Added few comments

**Comment by aayushijain1609** [comment]
Yes sir,
 I will do all the changes as soon as possible.

File: Assigment2/Assignment2.java (hunk 2 above)
Comment (aayushijain1609) [reply Assigment2/Assignment2.java:24]: > Can you please add a proper message for invalid?
Yes sir I specified each method .

File: Assigment2/assignment2/Cricle.java (hunk 3 above)
Comment (aayushijain1609) [reply Assigment2/assignment2/Cricle.java:15]: > Can you please throw an exception from this method?

yes sir . 
I did it .

**Pull request merged**